tables/Cuneiform-transliteration.utd

```diff
@@ -0,0 +1,186 @@
+
+# liblouis: Cuneiform-Transliteration
+#
+# ------------
+#-index-name: Cuneiform-Transliteration uncontracted
+#-display-name: Cuneiform-Transliteration uncontracted
```

Comment:
Could we call the table something else? The braille table itself does not transliterate anything. It handles already transliterated Cuneiform text, so something like "Transliterated Cuneiform braille" sounds more clear to me.

Note that you could in theory also have a braille table that converts Cuneiform signs to braille directly, using the transliteration method. Another example of such as table is zh-chn.ctb, which goes from simplified Chinese characters to braille through pinyin.